graph/typeDefs.graphql

```diff
@@ -339,6 +339,9 @@ type Comment {
 
   # describes how the comment can be edited
   editing: EditInfo
+
+  # Indicates if it has a parent
+  hasParent: Boolean
```

Comment:
@wyattjoh Would you say it makes more sense to expose `parent_id: String` instead? I think there were other occasions were we just needed the `parent_id` but used the `parent` endpoint which fetches the whole comment from mongodb.